src/amcl_wrapper.c

```diff
@@ -0,0 +1,50 @@
+#include <amcl/ecdh_NIST256.h>
```

Comment:
You should also include our copyright notice at the top of any file that's part of an open-source project. Just copy-paste the one from, say, `src/libsodium_wrapper.c`.